refresher/utils/mongo.go

```diff
@@ -2,9 +2,12 @@ package utils
 
 import (
 	"context"
+	"fmt"
```

Comment:
Customize your functions to receive a mongo client object instead of create one every time. If you feel Mongo client should expose those functions add them to go kit